docs/platforms/react-native/troubleshooting/index.mdx

```diff
@@ -15,78 +15,6 @@ This issue was fixed in version [4.1.1](https://github.com/getsentry/sentry-reac
 
 For more details, [see the troubleshooting page for the iOS SDK](/platforms/apple/troubleshooting/#wrong-app-start-data).
 
-## Unhandled Promise Rejections
```

Comment:
`l`: we could keep that section until [line 26](https://github.com/getsentry/sentry-docs/pull/14236/files#diff-bf9ff7aee17e39cc0295fbb822c534a823c8362f6f860dab00580d2017fb5993L26) and then link to the new page, wdyt?

Reply:
Good idea @kahest 👍 
Updated with 0abc8ead855c438f02b10e1aa2c344a2406dd5b1